text/convert.py

```diff
@@ -34,3 +37,125 @@ def run(self):
             for seq_tag, entry in d.items():
                 assert isinstance(entry, str), f"expected str, got {entry!r} (type {type(entry).__name__})"
                 out.write(entry + "\n")
+
+
+class TextDictToStmJob(Job):
+    """
+    Similar as :class:`CorpusToStmJob`, but does not use the Bliss XML, but instead the text dict as input
+    (e.g. via :class:`CorpusToTextDictJob`).
+    """
+
+    def __init__(
+        self,
+        text_dict: Path,
+        *,
+        exclude_non_speech: bool = True,
+        non_speech_tokens: Optional[List[str]] = None,
+        remove_punctuation: bool = True,
+        punctuation_tokens: Optional[Union[str, List[str]]] = None,
+        fix_whitespace: bool = True,
+        name: str = "",
```

Comment:
I don't think this name is ever used except to change the filename (on disk) of the output file.. which anyway should be irrelevant.
-> remove?

Reply:
Maybe it influences how it is shown in sclite? I don't really know.

But I checked, and I don't see any usage of `name` in `CorpusToStmJob` across i6_experiments (if PyCharm search is reliable here...), so maybe it's really rarely (or never) used, so then yea, we could remove this.